llvm/lib/Target/STM8/STM8ISelLowering.cpp

```diff
@@ -22,7 +22,7 @@ namespace llvm {
 
 STM8TargetLowering::STM8TargetLowering(const STM8TargetMachine &TM,
                                        const STM8Subtarget &STI)
-    : TargetLowering(TM), Subtarget(STI) {}
+    : TargetLowering(TM) {
```

Comment:
```suggestion
    : TargetLowering(TM) {}
```

Reply:
You're right. I would ignore that rn, because it happend due to me splitting up changes.